.github/workflows/rw-backup-postgres-database.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           -U ${{ inputs.database_user }} \
           -Fc -v --no-owner --no-privileges \
           ${{ inputs.options }} \
-          -f ${{ inputs.backup_file_name }} \
+          -f postgres_backup.sql \
```

Comment:
There should be no changes here as the maria db backup does not use this pipeline. Please revert the change

Reply:
Maybe we added this for consistency. If it is correct then we can keep the change